po/common/common.pot

```diff
@@ -5296,8 +5296,8 @@ msgid "Forgotten password?"
 msgstr "Mot de passe oublié ?"
 
 msgctxt "reset_password_error"
-msgid "We could not reinitialize your password, please contact us for assistance."
-msgstr "We could not reinitialize your password, please contact us for assistance."
+msgid "Your password could not be reset, please contact us for assistance."
+msgstr "Message shown if the password could not be reset by the system."
```

Comment:
msgstr should be empty, is this a comment?

Reply:
The msgstr has data everywhere. Should this be changed throughout the file?

Reply:
common.pot is the template, so msgstr should be empty in it.

Reply:
`.pot` = po template as opposed to `.po`